From 834f4290d396ffd5cb3d589476f75dd4baa7b6c2 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Mon, 31 Oct 2016 15:06:36 -0700
Subject: [PATCH 1006/1008] Fixup libstdc++ header search paths for older
 versions of Mac OS X

The subpaths were removed in Lion.  Leopard and Snow Leopard had darwin8
symlinks for compatibility.

https://llvm.org/bugs/show_bug.cgi?id=30858

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
---
 lib/Frontend/InitHeaderSearch.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git llvm_master/tools/clang/lib/Frontend/InitHeaderSearch.cpp macports_master/tools/clang/lib/Frontend/InitHeaderSearch.cpp
index 8a70404629..db49a73533 100644
--- llvm_master/tools/clang/lib/Frontend/InitHeaderSearch.cpp
+++ macports_master/tools/clang/lib/Frontend/InitHeaderSearch.cpp
@@ -375,7 +375,7 @@ void InitHeaderSearch::AddDefaultCPlusPlusIncludePaths(
                                                 "powerpc-apple-darwin10", "",
                                                 "ppc64", triple);
       IsBaseFound |= AddGnuCPlusPlusIncludePaths("/usr/include/c++/4.0.0",
-                                                 "powerpc-apple-darwin10", "",
+                                                 "powerpc-apple-darwin8", "",
                                                  "ppc64", triple);
       break;
 
@@ -385,7 +385,7 @@ void InitHeaderSearch::AddDefaultCPlusPlusIncludePaths(
                                                 "i686-apple-darwin10", "",
                                                 "x86_64", triple);
       IsBaseFound |= AddGnuCPlusPlusIncludePaths(
-          "/usr/include/c++/4.0.0", "i686-apple-darwin8", "", "", triple);
+          "/usr/include/c++/4.0.0", "i686-apple-darwin8", "", "x86_64", triple);
       break;
 
     case llvm::Triple::arm:
-- 
2.17.1 (Apple Git-112)

